drivers/SmartThings/philips-hue/src/handlers/commands.lua

```diff
@@ -177,6 +178,7 @@ end
 ---@param args table
 local function do_setHue_action(driver, device, args)
 
+  --Note: this will always be 0 due to state cache changes
```

Comment:
This actually seems like it could be a problem, because using a 0 for the other value will be a different color.  So I think we need to identify another solution than just not storing.

Reply:
Same issue here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1671/files#diff-ddccea1fd0c65608f45092f0f576c125a2afaebd37e7b9bb78855eece84037e5R197

My pitch is to capture the values in a field [here](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/drivers/SmartThings/philips-hue/src/handlers/attribute_emitters.lua#L26), either vol or nonvol

Reply:
Captured in a field.